packages/renderer/src/lib/feedback/SendFeedback.spec.ts

```diff
@@ -24,170 +24,47 @@ import { beforeAll, expect, test, vi } from 'vitest';
 
 import SendFeedback from './SendFeedback.svelte';
 
-// fake the window.events object
+const mocks = vi.hoisted(() => ({
+  gitHubIssueMock: vi.fn(),
+  developersFeedbackMock: vi.fn(),
+}));
```

Comment:
issue: please do not use vi.hoisted and mocks

vi.mock and vi.mocked is usually enough